Fix iOS local URL support (fixes #114)

rafern opened the pull request:
Adds a new WebviewScaffold parameter called localUrlScope which is the allowed directory for the URL and sub-resources. This way, local HTML files can read local resources as well, within the allowed directory set in the new parameter. The file URI scheme is now also allowed. If localUrlScope is null, then the old behaviour is used, in which only the local file is allowed and sub-resources are disallowed.

charafau commented:
hi @rafern thanks for PR. Would it be possible to also add note in README how to access that file? I assume that I need to add it to pubspec?

rafern commented:
I've added instructions on how to use the new option to README and fixed the conflicts. There shouldn't be anything else required in the pubspec unless you want to bump the version up. Feel free to ask for anything else.

charafau commented:
thanks
